pallets/automation-price/src/tests.rs

```diff
@@ -501,7 +501,7 @@ fn test_shift_tasks_movement_through_price_changes() {
 			asset2.to_vec(),
 			1000u128,
 			"gt".as_bytes().to_vec(),
-			vec!(100),
+			vec!(10100),
```

Comment:
Could we leverage variables to make these numbers more readable? For example, instead of 10100, we could use something like `target_price_1 + 100`.

Basically we just need to know how much higher or lower these numbers are compared to a target number in mock, right?